src/GitHubExtension/Widgets/GitHubRepositoryWidget.cs

```diff
@@ -69,16 +70,35 @@ public override void OnActionInvoked(WidgetActionInvokedArgs actionInvokedArgs)
 
     protected override void ResetWidgetInfoFromState()
     {
+        JsonNode? dataObject = null;
         try
         {
-            var dataObject = JsonNode.Parse(ConfigurationData);
-
-            if (dataObject == null)
+            dataObject = JsonNode.Parse(ConfigurationData);
+        }
+        catch (JsonException)

```

Comment:
Nit: Good idea to log this exception as well.

Reply:
Agree, and added!